pkg/storage/postgres/postgres.go

```diff
@@ -747,8 +772,7 @@ func (p *Postgres) ReadChanges(
 	return changes, contToken, nil
 }
 
-// IsReady reports whether this Postgres datastore instance is ready
-// to accept connections.
+// IsReady reports whether this Postgres datastore instance is ready to accept connections.
```

Comment:
```suggestion
// IsReady please see sqlcommon.IsReady
```

In my IDE it looks like this:

<img width="1019" alt="image" src="https://github.com/openfga/openfga/assets/5374887/13fce2a9-65dc-4f8c-aa21-6bee387dd7e2">

Reply:
This isn't a matter of IDE, but of `godoc`. If we change the comment to what you suggest the godoc will display the following, without any link:

<img width="973" alt="image" src="https://github.com/openfga/openfga/assets/28300158/c1fd6e64-4560-4d4a-adbf-868c9556e433">